packages/react-sdk/src/components/ElementBar/FontFamilyButton/FontFamilyButton.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2025 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MenuItem, Select } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFontFamily } from '../../../lib/text-formatting/useFontFamily';
+import { useActiveElements, useElements } from '../../../state';
+import { TextFontFamily } from '../../../state/crdt/documents/elements';
+
+const FONT_FAMILY = [
+  'inter',
```

Comment:
For me it would feel more natural to put „Inter“ at it's alphabetical position into the list. Wdyt?

Reply:
Inter sorted into the list